docs/content/development/extensions-core/kinesis-ingestion.md

```diff
@@ -126,37 +126,37 @@ A sample supervisor spec is shown below:
+| `indexSpec`                           | Object         | Tune how data is indexed. See [IndexSpec](#indexspec) for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | no                                                                                                           |
+| `indexSpecForIntermediatePersists`    |                | Defines segment storage format options to be used at indexing time for intermediate persisted temporary segments. this can be used to disable dimension/metric compression on intermediate segments to reduce memory required for final merging. however, disabling compression on intermediate segments might increase page cache use while they are used before getting merged into final segment published, see [IndexSpec](#indexspec) for possible values.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | no (default = same as indexSpec)                                                                             |
```

Comment:
The first letter of the other sentences can also be capitalized ("This" and "However").

Reply:
Dang, thanks!